sdk/batch/Azure.ResourceManager.Batch/api/Azure.ResourceManager.Batch.netstandard2.0.cs

```diff
@@ -1202,4 +1208,43 @@ public enum NodeCommunicationMode
         Classic = 1,
         Simplified = 2,
     }
+    public partial class OSDisk
+    {
+        public OSDisk() { }
+        public Azure.ResourceManager.Batch.Models.BatchDiskCachingType? Caching { get { throw null; } set { } }
+        public int? DiskSizeGB { get { throw null; } set { } }
+        public Azure.ResourceManager.Batch.Models.BatchDiffDiskPlacement? EphemeralOSDiskPlacement { get { throw null; } set { } }
+        public Azure.ResourceManager.Batch.Models.BatchStorageAccountType? ManagedDiskStorageAccountType { get { throw null; } set { } }
+        public bool? WriteAcceleratorEnabled { get { throw null; } set { } }
+    }
+    public partial class SecurityProfile
+    {
+        public SecurityProfile() { }
+        public bool? EncryptionAtHost { get { throw null; } set { } }
+        public Azure.ResourceManager.Batch.Models.SecurityType? SecurityType { get { throw null; } set { } }
+        public Azure.ResourceManager.Batch.Models.UefiSettings UefiSettings { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct SecurityType : System.IEquatable<Azure.ResourceManager.Batch.Models.SecurityType>
```

Comment:
same here, could we have it as `BatchSecurityType`?

Reply:
updated